src/app/about/about.module.css

```
@@ -676,20 +645,21 @@
     box-shadow 160ms ease,
     transform 120ms ease;
 
-  box-shadow: 0 4px 14px var(--button-shadow, rgba(0, 0, 0, 0.12)); /* ✅ 더 선명 */
+  box-shadow: 0 4px 14px var(--button-shadow, rgba(0, 0, 0, 0.12));
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)